javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
```

Comment:
The `useFormalName` variable should be the parameter of the function. The function then looks like:
```javascript
function getFullName(firstname, surname, useFormalName) {
  ...
}
```
If you add this `useFormalName` parameter, the function you already wrote will work exactly how you want it, when you call it like `let fullname = getFullname("Mark", "Twain", true);` for example.

The parameter names `fullname1` and `fullname2` could be renamed to `firstname` and `surname` since these names explain what these parameters represent better. 🧠 

Also, make sure to check if either the `firstname` or the `surname` is an empty string, and return some message that this is not correct behaviour. ☝️